src/api/v2/feedback.py

```diff
@@ -0,0 +1,89 @@
+import asyncio
+
+# import logging
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, HTTPException, Query, status
+from pydantic.fields import Field
+
+from src.app.models.feedback import Feedback
+from src.app.views.input.feedback import FeedbackInput
+from src.app.views.response.feedback import Feedback as FeedbackResponse
+from src.app.views.response.feedback import FeedbackCount as FeedbackCountResponse
+from src.app.views.response.ok import Ok
+from src.core.fastapi.dependencies.session import get_session
+from src.core.fastapi.dependencies.to_jagex_name import to_jagex_name
+
+router = APIRouter(tags=["feedback"])
+
+# logger = logging.getLogger(__name__)
+
+
+@router.get("/players/feedback", response_model=list[FeedbackResponse])
+async def get_feedback(
+    name: list[Annotated[str, Field(..., min_length=1, max_length=13)]] = Query(
+        ...,
+        min_length=1,
+        description="Name of the player",
+        example=["Player1", "Player2"],
+    ),
+    session=Depends(get_session),
+):
+    """
+    Post feedback data for a user.
+
+    Args:
+        feedback (FeedbackIn): The feedback data to post.
+
+    Returns:
+        FeedbackOut: An object containing the posted feedback data.
+
+    Raises:
+        HTTPException: Returns a 404 error with the message "Player not found" if no data is found for the user.
+
+    """
+    feedback = Feedback(session)
+    names = await asyncio.gather(*[to_jagex_name(n) for n in name])
+    data = await feedback.get_feedback(player_names=names)
+
+    return data
```

Comment:
we may not want to publicly expose this

Reply:
so for the private api?

Reply:
maybe in the future but probably not tbh

Reply:
where would we stash it?